src/test/java/org/opensearch/geospatial/GeospatialRestTestCase.java

```diff
@@ -119,4 +120,15 @@ public String buildGeoJSONFeatureAsString(String type, Object value, Map<String,
         final Map<String, Object> docMap = (Map<String, Object>) responseMap.get(SOURCE);
         return docMap;
     }
+
+    protected String convertToString(Map<String, Object> contentMap) {
+        JSONObject json = new JSONObject();
```

Comment:
Why use JSONObject as opposed to XContentFactory.jsonBuilder()?

Reply:
I was thinking about it for long time and choose JSONObject because it is simpler to build than XContentFactory.jsonBuilder() and i added dependency as testImplementation so it should not affect performance. Do you see any problem in this approach?